src/Workspaces/VisualBasicTest/VisualBasicSyntaxFactsServiceTests.vb

```diff
@@ -530,6 +530,24 @@ End Class"
 
             Return service.IsQueryKeyword(token)
         End Function
+
+        <Fact, WorkItem(40917, "https://github.com/dotnet/roslyn/issues/40917")>
+        Public Sub IsLeftSideOfCompoundAssignment()
+            Assert.True(IsLeftSideOfCompoundAssignment(WrapInMethod("
+Dim index As Integer = 0
+$$index += 1")))
+        End Sub
+
+        Private Function IsLeftSideOfCompoundAssignment(markup As String) As Boolean
+            Dim code As String = Nothing
+            Dim position As Integer
+            MarkupTestFile.GetPosition(markup, code, position)
+            Dim tree = SyntaxFactory.ParseSyntaxTree(code)
```

Comment:
Would `code` still be `Nothing` here or does `MarkupTestFile.GetPosition` populate `code` as well as `position`?

Reply:
It populates both code and position.